src/main/webapp/assets/css/feedStyle.css

```diff
@@ -212,13 +207,39 @@ select {
   color: #404040;
 }
 
+#modal-submitted {
```

Comment:
Love this!!

Reply:
Thanks, it was a good idea! Thanks for creating the issue